flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/RecordWriterOutput.java

```diff
@@ -72,6 +76,11 @@ public RecordWriterOutput(
 
```

Comment:
We can't throw, because that would crash the program. This is a good catch, though! I will remove the duplication.